picard/file.py

```diff
@@ -395,12 +401,13 @@ def update(self, signal=True):
         names.update(self.orig_metadata.keys())
         clear_existing_tags = config.setting["clear_existing_tags"]
         for name in names:
-            if not name.startswith('~') and self.supports_tag(name):
+            if self.supports_tag(name):
                 new_values = new_metadata.getall(name)
                 if not (new_values or clear_existing_tags):
                     continue
                 orig_values = self.orig_metadata.getall(name)
                 if orig_values != new_values:
+                    print "Tag",name,"has changed."
```

Comment:
The print statement was for debugging and should be removed. I am uncertain why albumrating should be considered a change, and will try to work out what is causing this.